ocaml/xapi/helpers.ml

```diff
@@ -793,8 +793,7 @@ let max_version_in_pool : __context:Context.t -> int list =
   in
   List.fold_left max_version [] versions
 
-let string_of_int_list =
-  List.fold_left (fun acc x -> Printf.sprintf "%s.%s" (string_of_int x) acc) ""
+let string_of_int_list xs = List.map string_of_int xs |> String.concat ""
```

Comment:
The final string should be `"."`.

Reply:
I checked again, this function is unused:

```
$ rg string_of_int_list
ocaml/xapi/helpers.ml
796:let rec string_of_int_list : int list -> string = function
803:        string_of_int x ^ "." ^ string_of_int_list xs
```

Reply:
third time lucky..